Tools/README

```diff
@@ -1,10 +1,20 @@
 This directory contains a number of Python programs that are useful
 while building or extending Python.
 
+.build          Automatically generated directory by the build system
+                contain  build artifacts and intermediate files. 
+
 buildbot        Batchfiles for running on Windows buildbot workers.
 
+c-analyzer      CPython's C code makes extensive use of global variables.
+
+cases_generator A Python tool for providing test cases for Python's parser module.
+
 ccbench         A Python threads-based concurrency benchmark. (*)
 
+clinic          Tool for generating C code that interfaces with Python's 
+                C API. 
```

Comment:
This is the Argument Clinic tool, can we summarise from the first two lines of the abstract at https://docs.python.org/3/howto/clinic.html?

Reply:
Can you confirm if it's ok for this suggestion ? In order to see if i commit with the others changes.

Thank you.

Reply:
Looks good, here's a small tweak:

A preprocessor for CPython C files in order to automate the boilerplate involved with writing argument parsing code for "builtins".